src/components/schemas/report.yaml

```diff
@@ -353,7 +353,7 @@ spender_report_in:
     purpose:
       type: string
       description: |
-        Indicates purpose of the reports.
+        Indicates purpose of the reports. It can only be updated for an existing report before it gets `APPROVED`.
```

Comment:
Reword to https://github.com/fylein/fyle-platform-docs/pull/327/files#r1279122472